libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -70,15 +103,20 @@ export const fetchAccount = async (
   blockHeight?: number;
   balance: BigNumber;
   spendableBalance: BigNumber;
+  assets: BalanceAsset[];
 }> => {
   let account: typeof AccountRecord = {};
   let balance: Record<string, any> = {};
+  let assets: BalanceAsset[] = [];
 
   try {
     account = await server.accounts().accountId(addr).call();
-    balance = account.balances.find((balance) => {
+    balance = account.balances?.find((balance) => {
       return balance.asset_type === "native";
     });
+    assets = account.balances?.filter((balance) => {
```

Comment:
Is this only assets that are defined or is it all the assets that an account have ?

Reply:
These are all assets that the account has. We're checking against the list of supported Ledger Live assets in `stellar/tokens.ts/buildSubAccounts()` method. I'll add a comment for clarity.